samples/backend/python/quart/.env

```diff
@@ -0,0 +1,5 @@
+PORT=3000
```

Comment:
For our samples, we usually only put a .env.sample, and then we make developers manually make a .env based on that, with .env in the .gitignore. That's how we try to reduce the amount of secrets checked in.

Probably not a big deal for this repo, once again, since am not sure if folks will be cloning and running, but FYI about our security best practices.

Fortunately there's no secrets here anyway, as you're presumably assuming managed identity.

Reply:
Will update to do that, that seems like a better practice in general